balrog.wsgi

```diff
@@ -1,8 +1,20 @@
 import logging
 from os import path
+from os import getenv
 import site
 import sys
 
+try:
+    import newrelic.agent
```

Comment:
I'd kinda rather not at this point. These changes are intended to pull in New Relic support if it's available, not require it to be available. In our production environment, it is, so it gets imported. However on a developer's laptop, it may not be (and even if it was, they wouldn't necessarily have a valid license key to use with it), and I don't think we should require it. It's inherently an optional component.

If we do want to require it, I think requirements/prod.txt is probably a better place- signifying that it should exist for production, but not is not critical for other environments.